### ISSUE TITLE
modStaticResource's class is missing relationship to modResourceGroupResource

### ISSUE THREAD

**Issue by goldsky** [modxcms/revolution#12728]
When a resource group is applied to a static resource, this error occurs:

```
[2015-10-19 10:49:22] (ERROR @ /home/***/core/xpdo/om/xpdoobject.class.php : 2548) PHP warning: Invalid argument supplied for foreach()
[2015-10-19 10:49:22] (ERROR @ /index.php) No foreign key definition for parentClass: modStaticResource using relation alias: modResourceGroupResource
```

**Comment by sottwell** [comment]
While you're at it, maybe give them support for media sources, too.

**Comment by alroniks** [comment]
I can not reproduce. I guess it was fixed during the last 5.5 years.
